https://github.com/mlflow/mlflow/pull/3552/checks?check_run_id=1378317395#step:4:100

```
>       np.testing.assert_allclose(actual_scoring_response, expected, rtol=3e-6)
E       AssertionError: 
E       Not equal to tolerance rtol=3e-06, atol=0
E       
E       Mismatched elements: 1 / 150 (0.667%)
E       Max absolute difference: 1.1920929e-07
E       Max relative difference: 3.9720207e-06
E        x: array([[ 1.651972],
E              [ 1.536899],
E              [ 1.5181  ],...
E        y: array([[ 1.651972],
E              [ 1.536899],
E              [ 1.5181  ],...
```

Is `rtol=3e-6` too strict?

--- a/tests/keras/test_keras_model_export.py
+++ b/tests/keras/test_keras_model_export.py
@@ -555,7 +555,7 @@ def test_save_and_load_model_with_tf_save_format(tf_keras_model, model_path):
 
 @pytest.mark.large
 def test_load_without_save_format(tf_keras_model, model_path):
-    """Ensures that keras models with save_format can still be loaded."""
+    """Ensures that keras models without save_format can still be loaded."""
     mlflow.keras.save_model(tf_keras_model, model_path, save_format="h5")
     model_conf_path = os.path.join(model_path, "MLmodel")
     model_conf = Model.load(model_conf_path)

--- a/tests/keras/test_keras_model_export.py
+++ b/tests/keras/test_keras_model_export.py
@@ -249,7 +249,7 @@ def test_model_save_load(build_model, save_format, model_path, data):
     actual_scoring_response = pd.read_json(
         scoring_response.content, orient="records", encoding="utf8"
     ).values.astype(np.float32)
-    np.testing.assert_allclose(actual_scoring_response, expected, rtol=3e-6)
+    np.testing.assert_allclose(actual_scoring_response, expected, rtol=1e-5)
 
     # test spark udf
     spark_udf_preds = score_model_as_udf(
@@ -555,7 +555,7 @@ def test_save_and_load_model_with_tf_save_format(tf_keras_model, model_path):
 
 @pytest.mark.large
 def test_load_without_save_format(tf_keras_model, model_path):
-    """Ensures that keras models saved in earlier versions (before save_format support) can still be loaded."""
+    """Ensures that keras models with save_format can still be loaded."""
     mlflow.keras.save_model(tf_keras_model, model_path, save_format="h5")
     model_conf_path = os.path.join(model_path, "MLmodel")
     model_conf = Model.load(model_conf_path)

